app/javascript/react/components/PizzaShowContainer.js

```diff
@@ -0,0 +1,40 @@
+import React, { useState, useEffect } from 'react'
+
+const PizzaShowContainer = (props) => {
+    const [pizza, setPizza] = useState({
+        brand: "",
+        product_name: "",
+        cost: ""
+    })
+
+    let pizzaId = props.match.params.id
+
+    const getPizza = async () => {
+	    try {
+	      const response = await fetch(`/api/v1/pizzas/${pizzaId}`)
+	      if (!response){
+	        const errorMessage = `${response.status} (${response.statusTest})`
+	        const error = new Error(errorMessage)
+	        throw(error)
+	      }
+	      const parsedPizza= await response.json()
+	      setPizza(parsedPizza)
+	    } catch(err){
+	      console.error(`Error in fetch: ${err.message}`)
+	    }
+    }  
+    
+    useEffect(()=>{
```

Comment:
These look indented a bit more than the other files. Make sure everyone has the same settings on their VScode. Best practice is an indent of 2.